src/Products/PloneKeywordManager/tool.py

```diff
@@ -138,34 +138,36 @@ def updateObject(self, obj, indexName, value):
     def getKeywords(self, indexName="Subject"):
         processQueue()
         if indexName not in self.getKeywordIndexes():
-            raise ValueError("%s is not a valid field" % indexName)
+            raise ValueError(f"{indexName} is not a valid field")
```

Comment:
Python is weird here.

Line 50 {indexName} is a set
Line 141 {indexName} is an f-string expression.

I think I liked the 'set()' syntax better.  It's more readable and explicit.   But that's just me.

What do you think?

Reply:
Just  a bit weird. Both is valid. I don't mind.